sources/api/migration/migrations/v1.8.0/boot-setting-metadata/src/main.rs

```diff
@@ -0,0 +1,23 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::{AddMetadataMigration, SettingMetadata};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added a new setting and generator for kernel boot configuration
+fn run() -> Result<()> {
+    migrate(AddMetadataMigration(&[SettingMetadata {
```

Comment:
It is not possible unless we introduce a new migration helper that does both within the same forward/backward migration. Each migration can only have a single `migrate` invocation. Because we only write out the datastore once at the end hen the migration binary returns. Any subsequent calls within the same migration binary invocation will just overwrite what was migrated previously.